From 29252c7934fb5f9ffd2c3929416ee84a1d1d9ff2 Mon Sep 17 00:00:00 2001
From: walterzhaoJR <519362600@qq.com>
Date: Sat, 30 Oct 2021 13:52:49 +0000
Subject: [PATCH 1/2] update a

---
 a | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/a b/a
index 5581a17..58039ac 100644
--- a/a
+++ b/a
@@ -1,5 +1,5 @@
 This is the file a;
-add a line
+nnnadd a line
 add a new line
 merge2
 merge1
-- 
1.8.3.1

